trace-sdk/src/test/java/io/bitrise/trace/data/management/formatter/device/DeviceCarrierDataFormatterTest.java

```diff
@@ -0,0 +1,29 @@
+package io.bitrise.trace.data.management.formatter.device;
+
+import org.junit.Test;
+
+import io.bitrise.trace.data.collector.device.DeviceCarrierDataCollector;
+import io.bitrise.trace.data.dto.Data;
+import io.bitrise.trace.data.dto.FormattedData;
+import io.bitrise.trace.data.management.formatter.BaseDataFormatterTest;
+import io.bitrise.trace.data.resource.ResourceEntity;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for {@link DeviceCarrierDataFormatter}.
+ */
+public class DeviceCarrierDataFormatterTest extends BaseDataFormatterTest {
+
+    final String carrier = "vodafone";
```

Comment:
nit: private visibility is enough for this variable.

Reply:
So i thought that too - but AS gives the following warning: 
<img width="738" alt="Screenshot 2021-05-10 at 15 14 47" src="https://user-images.githubusercontent.com/71286749/117673215-7e7ef580-b1a2-11eb-836f-91cbcf5de754.png">

I figured the warnings would be more annoying than the variable not being private. What do you think?

Reply:
As it is used only in one place, lint is right, it can be a local variable.

Reply:
Have updated, and updated all the other references too (let's keep the conversation in one thread?)